src/drivers/cuda/driver_cublasLt.c

```diff
@@ -0,0 +1,76 @@
+/* StarPU --- Runtime system for heterogeneous multicore architectures.
+ *
+ * Copyright (C) 2009-2024  University of Bordeaux, CNRS (LaBRI UMR 5800), Inria
+ *
+ * StarPU is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or (at
+ * your option) any later version.
+ *
+ * StarPU is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU Lesser General Public License in COPYING.LGPL for more details.
+ */
+
+#include <common/config.h>
+
+#include <starpu.h>
+#include <starpu_cuda.h>
+#include <core/workers.h>
+
+#ifdef STARPU_HAVE_LIBCUBLASLT
+#include <cublasLt.h>
+
+static cublasLtHandle_t cublasLt_handles[STARPU_NMAXWORKERS];
+static cublasLtHandle_t main_handle;
+
+static void init_cublasLt_func(void *args STARPU_ATTRIBUTE_UNUSED)
+{
+	cublasLtCreate(&cublasLt_handles[starpu_worker_get_id_check()]);
+	// No need for setting streams, because the cublasLt handles are not along with streams
```

Comment:
Mmm, but AIUI the user should call `starpu_cublas_init` for streams to be properly set up in cublas, for cublasLt to benefit from them? This should really be documented in the starpu_cublasLt_init documentation in starpu_cublasLt.h

Reply:
Please refer to [this example](https://docs.nvidia.com/cuda/cublas/#cublasltmatmul). When we call cublasLt functions, we need to pass a stream explicitly.

Reply:
Ok, it will be useful to document that users should use `starpu_cuda_get_local_stream()` for that

Reply:
where should I add docs for that? `starpu_cublasLt.h`?

Reply:
Better document it both in `starpu_cublasLt.h` for `starpu_cublasLt_init` , and in `check_list_performance.doxy`, along documenting the availability of the helpers.

Reply:
I add some descriptions to `doc/doxygen/chapters/starpu_faq/check_list_performance.doxy`